packages/editor/src/client/locales/en.json

```diff
@@ -55,7 +55,8 @@
         "draft": "Draft",
         "pending": "Pending",
         "published": "Published",
-        "publishDate": "Publish Date",
+        "publishTime": "Publish Time",
```

Comment:
why did you rename this? It's still `Publish Date & Time`

Reply:
Was a visual consideration as well as placing the time picker before the date, but if the more common usage is to set the date then I'll change them both back.